tools/dm/skip-replace-sqls.md

```diff
@@ -0,0 +1,608 @@
+        - Common spaces are not supported in the regular expression temporarily. You can replace the space with `\s` or `\s+` if it is needed.
+        - The regular expression must be prefixed with `~`. For details, see [regular expression syntax](https://golang.org/pkg/regexp/syntax/#hdr-Syntax).
+        - The schema/table name in the regular expression must be converted by the optional router-rule, so the converted name is consistent with the target schema/table name in the downstream. For example, if there are ``` `shard_db_1`.`shard_tbl_1` ``` in the upstream and ``` `shard_db`.`shard_tbl` ``` in the downstream, then you should match ``` `shard_db`.`shard_tbl` ```.
+        - The schema/table/column name in the regular expression should be marked by ``` ` ```, for example, ``` `db1`.`tbl1` ```.
+
++ `sharding`: 
+    - Flag parameter, boolean, `--sharding`
+    - If `--worker` is not specified, `sharding` is required; if `--worker` is specified, `sharding` is forbidden to use.
+    - If `sharding` is specified, it indicates that the presetted operation is going to be executed in the DDL lock owner during the sharding DDL replication.
+
++ `task-name`: 
+    - Non-flag parameter, string, required
+    - `task-name` specifies the name of the task in which the presetted operation is going to be executed. 
+
+#### sql-replace
+
+`sql-replace` allows you to preset a replace operation that is to be executed when the position or the SQL statement of the binlog event matches with the specified `binlog-pos` or `sql-pattern`.
+
+##### Command usage
+
+```bash
+sql-replace <--worker=127.0.0.1:8262> [--binlog-pos=mysql-bin|000001.000003:3270] [--sql-pattern=~(?i)ALTER\s+TABLE\s+`db1`.`tbl1`\s+ADD\s+COLUMN\s+col1\s+INT] [--sharding] <task-name> <SQL-1;SQL-2>
+```
+
+##### Arguments description
+
++ `worker`: 
+    - same with `--worker` of `sql-skip`
+
++ `binlog-pos`: 
+    - same with `--binlog-pos` of `sql-skip`
+
++ `sql-pattern`: 
+    - same with `--sql-pattern` of `sql-skip`
+
++ `sharding`: 
+    - same with `--sharding` of `sql-skip`
+
++ `task-name`: 
+    - same with `task-name` of `sql-skip`
+
++ `SQLs`: 
+    - Non-flag parameter, string, required 
+    - `SQLs` specifies the new SQL statements that are going to replace the original binlog event. You should separate multiple SQL statements with `;`, for example, ``` ALTER TABLE shard_db.shard_table drop index idx_c2;ALTER TABLE shard_db.shard_table DROP COLUMN c2; ```. 
+
+### Usage examples
+
+#### Passively skip after the replication gets interrupted
+
+##### Business scenario
+
+Assume that we need to replicate the upstream table `db1.tbl1` to the downstream TiDB (not in the scenario of merging and replicating data from sharded tables). The initial table schema is:
```

Comment:
```suggestion
Assume that you need to replicate the upstream table `db1.tbl1` to the downstream TiDB (not in the scenario of merging and replicating data from sharded tables). The initial table schema is:
```

Reply:
Generally, use "you" instead of "we" in the documentation. Please check and update similar expressions in this PR. Thanks!

Reply:
Comment addressed. PTAL again. @lilin90